src/dataflow/src/logging/materialized.rs

```diff
@@ -38,6 +38,46 @@ pub enum MaterializedEvent {
         /// Globally unique identifier for the source on which the dataflow depends.
         source: GlobalId,
     },
+    /// Tracks RTT statistics for a Kafka broker, by consumer
+    /// Reference: https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md
+    KafkaBrokerRTT {
+        /// Kafka name for the consumer
+        consumer_name: String,
+        /// Materialize source identifier
+        source_id: SourceInstanceId,
+        /// The Kafka broker for these metrics (may be multiple per consumer)
+        broker_name: String,
+        /// Splatted metrics from the rdkafka::statistics::Window struct
+        /// Window cannot be used as it does not satisfy several of the traits required
```

Comment:
should these two lines be a regular (not-doc) comment? It doesn't seem like they are meant to apply to the `min` field.

Reply:
I moved this comment to be on the struct itself!